website/package.json

```diff
@@ -63,5 +63,5 @@
     "remark-toc": "9.0.0",
     "remark-wiki-link": "2.0.1"
   },
-  "packageManager": "bun@1.1.21"
+  "packageManager": "bun@1.1.27"
 }
```

Comment:
I also tried to update to docusaurus to 3.5, but getting ESM errors. Looking into it.